test/e2e/prometheus_test.go

```diff
@@ -84,3 +85,101 @@ func TestPrometheusVolumeClaim(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestPrometheusAlertmanagerAntiAffinity(t *testing.T) {
+	pods, err := f.KubeClient.CoreV1().Pods(f.Ns).List(metav1.ListOptions{FieldSelector: "status.phase=Running"})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var alm = `affinity:
+	   podAntiAffinity:
+	      preferredDuringSchedulingIgnoredDuringExecution:
+	      - podAffinityTerm:
+	          labelSelector:
+	            matchExpressions:
+	            - key: alertmanager
+	              operator: In
+	              values:
+	              - main`
+
+	var k8s = `affinity:
+	 podAntiAffinity:
+	      preferredDuringSchedulingIgnoredDuringExecution:
+	      - podAffinityTerm:
+	          labelSelector:
+	            matchExpressions:
+	            - key: prometheus
+	              operator: In
+	              values:
+				  - k8s`
```

Comment:
nit: this indentation looks odd. can you use spaces only in this line?

Reply:
Of course will have to correct this line and use only spaces before symbols.
Thank you.

Reply:
Fixed